content/en/01-about-pixie/04-faq.md

```diff
@@ -43,47 +45,73 @@ order: 4
 
-#### Can I completely self host Pixie?
+### Can I completely self host Pixie?
 
 Yes. See the self-hosted [Install Guide](/installing-pixie/install-guides/self-hosted-pixie) to get started.
 
-#### Does Pixie offer a hosted cloud offering?
+### Does Pixie offer a hosted cloud offering?
 
-Yes. Pixie Community Cloud is a hosted version of Pixie. Pixie stores its data on the cluster, resulting in a very small cloud footprint which allows Pixie to offer the Community Cloud offering 100% free for all Pixie users. See the Community Cloud [Install Guide](/installing-pixie/install-guides/community-cloud-for-pixie) to get started.
+Yes. Pixie Community Cloud is a hosted version of Pixie. Pixie stores all data on the customer's cluster, resulting in a very small cloud footprint which allows Pixie to offer the Community Cloud offering 100% free for all Pixie users. See the Community Cloud [Install Guide](/installing-pixie/install-guides/community-cloud-for-pixie) to get started.
 
 ## Data Collection
 
-#### Where does Pixie store its data? How much data is stored?
+### Where does Pixie store its data?
 
-Pixie stores the data it collects on the cluster in-memory; data is not sent to any centralized backend outside of the cluster.
+Pixie stores the data it collects in-memory on the nodes in your cluster; no data is sent to a centralized backend outside of the cluster. This is true for both self-hosted Pixie and [Pixie Community Cloud](/about-pixie/faq#does-pixie-offer-a-hosted-cloud-offering).
+
+Pixie has a [2GiB memory requirement](/installing-pixie/requirements/#memory) per node. After installing Pixie, it is normal to see a temporary increase in memory usage of the `vizier-pem` pods as they begin to fill their data tables.
+
+### How much data does Pixie store?
 
 Retention time depends on the level of traffic in your cluster, but will generally be on the order of hours. Each data table in Pixie (e.g. `http_events`) has its own maximum size. These tables collect data until the maximum size is reached, and then begin dropping the oldest data.
 
-We recommend integrating with third-party observability tools (such as New Relic’s integration) to provide long-term retention.
+We recommend integrating with third-party observability tools (such as [New Relic’s integration](https://newrelic.com/platform/kubernetes-pixie)) to provide long-term retention.
+
+### How does Pixie secure its data?
+
+Pixie stores the telemetry data it collects in-memory on the nodes in your cluster. Data processing and script execution are also performed in the cluster. This is true for both self-hosted Pixie and [Pixie Community Cloud](/about-pixie/faq#does-pixie-offer-a-hosted-cloud-offering).
+
+End-to-end encryption is offered for data in flight between in-cluster storage and presentation in the UI / CLI / API:
```

Comment:
@vihangm - did you see this section? Should I update the Live UI / CLI's enabled statuses to reflect the current state or the intended near-future state?